server/controllers/auctions.js

```diff
@@ -36,6 +36,17 @@ router.post('/', (req, res) => {
   });
 });
 
+router.post('/ongoing', (req, res) => {
+  const { user } = req.body;
+  model.getUserBiddingAuctions(user)
+  .then((auctions) => {
+    res.status(200).json(auctions);
+  })
+  .catch((err) => {
+    res.status(500).send(err);
```

Comment:
well if a user doesn't have any ongoing auctions they will receive an empty array. The only way they it catch is if there is an error in processing the query, probably caused by an invalid userId. Not sure if 4xx is right because the user isn't controlling that aspect of the request. Let me know what you think.